test/smoke/package/src/helpers/AutomationHelper.ts

```diff
@@ -224,4 +224,21 @@ export default class AutomationHelper {
             catchFun,
         );
     }
+
+    public async addConfigurationWithRetry(
+        retryCount: number = 3,
+        pollRetryCount: number = 10,
+        pollRetryInterval: number = 1000,
+    ): Promise<void> {
+        const fun = async () => {
+            await this.app.workbench.debug.addConfiguration();
+            await this.app.workbench.quickinput.waitForQuickInputOpened(pollRetryCount);
+        };
```

Comment:
Maybe we may call it more specific like `addConfigurationOpenQuickInput` or simply `func`?